src/client/clientlauncher.cpp

```diff
@@ -59,7 +59,8 @@ ClientLauncher::~ClientLauncher()
 
 	delete g_fontengine;
 
-	device->drop();
+	if (device)
```

Comment:
Your feedback on the use of a command like “`git rebase --interactive`” indicates some software development difficulties which are known to some degree already.
We have got different opinions about “commit noise”.